airbyte-integrations/connectors/destination-azure-blob-storage/src/main/java/io/airbyte/integrations/destination/azure_blob_storage/AzureBlobStorageDestinationConfig.java

```diff
@@ -15,18 +15,26 @@
   private final String accountName;
   private final String accountKey;
   private final String containerName;
+  private final int outputStreamBufferSize;
   private final AzureBlobStorageFormatConfig formatConfig;
 
   public AzureBlobStorageDestinationConfig(
                                            final String endpointUrl,
                                            final String accountName,
                                            final String accountKey,
                                            final String containerName,
+                                           final int outputStreamBufferSize,
                                            final AzureBlobStorageFormatConfig formatConfig) {
     this.endpointUrl = endpointUrl;
     this.accountName = accountName;
     this.accountKey = accountKey;
     this.containerName = containerName;
+    if (outputStreamBufferSize > 0) {
```

Comment:
Could you please add some tests for it? Thanks

Reply:
A test at this line? I'm not sure I understand the ask. I think now with the spec having the min/max that I wasn't aware of previously this is largely unnecessary. The method here can only get an int, so we'll know it'll be 0 is nothing else? This was just to handle I someone put 0, or less than 0, in the spec. I'm sure technically someone could force inject an invalid value into the spec though.

Reply:
Sorry, I meant to add a test to make sure that we can successfully process this argument. Currently, all of our tests run with the default option. This new test would be also used to find potential regression issues during implementation in future

Reply:
Ah that makes sense, I'll try to dig through the tests later to set something up

Reply:
Added a new test for the output buffer size